### PR TITLE
Add info about donating to NumPy to contribute.md

### PR DESCRIPTION
**Pull request by InessaPawson**
Add info about donating to NumPy to contribute.md
Resolves https://github.com/numpy/numpy.org/issues/498

Improve Fundraising section

### CODE REVIEW

**Comment by stefanv** [comment]
I wonder if we should encourage small donations. Ultimately, recurring larger contributions (eg Tidelift), grants, etc. allow us to hire contractors. But I feel bad that we should fund-raise small amounts from people who don't have a lot. Better perhaps for them to help out in other ways?

File: content/en/contribute.md
Comment (rgommers) [inline content/en/contribute.md:110]: It would be better to have one sentence here linking to `about.md#donate` (see https://numpy.org/about/#donate), rather than duplicate this content.

Comment (rgommers) [reply content/en/contribute.md:110]: That content on the About page also needs a small update, to point to Open Collective (as pointed out in https://github.com/numpy/numpy.org/issues/498#issuecomment-1295930356)

**Review by rgommers — COMMENTED** [review]
Thanks Inessa!

> I wonder if we should encourage small donations. Ultimately, recurring larger contributions (eg Tidelift), grants, etc. allow us to hire contractors. But I feel bad that we should fund-raise small amounts from people who don't have a lot. Better perhaps for them to help out in other ways?

I can see arguments for this either way. However, this PR is more a minor maintenance update than a significant change, since we already have this content on the About page.

On the one hand, at the moment we don't have needs that we cannot cover with the ~$80,000 project balance (see https://opencollective.com/numpy). The one source of recurring unrestricted income for the project is $1,000/month from Tidelift (and that amount has been declining over time).

On the other hand, I can imagine that the good times (multiple maintainers getting significant day job time from their employers, as well as always >=1 active grant since 2017) won't last forever. And grant funding tends to not be unrestricted (luckily quite a bit of employer day job time is).

* * *

I'd merge this PR after comments are addressed, because it doesn't add anything that wasn't already there.

File: content/en/contribute.md
Comment (InessaPawson) [reply content/en/contribute.md:110]: @rgommers Thank you for your feedback! The button in `about.md#donate` is currently pointing to https://opencollective.com/numpy#category-CONTRIBUTE Did you have a different "destination" in mind?

Comment (rgommers) [reply content/en/contribute.md:110]: Sorry for the noise, the link is fine. I was either looking at an old page or had some temporary blindness - that comment was addressed a long time ago. Please ignore this part.

**Review by rgommers — APPROVED** [review]
LGTM now, in it goes. Thanks Inessa! And thanks for the review Stefan.

**Pull request merged**
